### DISCUSSION TITLE
Current symlink and nvm use

### DISCUSSION THREAD

**Discussion by PiousVenom** [coreybutler/nvm-windows#805] [Support]
I'm using `export NVM_SYMLINK_CURRENT=true` for use in my IDE. The problem I'm experiencing is whenever I change node versions, I get an error `rm: cannot remove ~/.nvm/current: Is a directory`. Is there something I've missed in this, or do I need to manually delete the `current` every time?

**Comment by coreybutler** [comment]
`NVM_SYMLINK_CURRENT` is not a supported environment variable. `NVM_SYMLINK` is. On Windows, normally `set` is used instead of `export` for env variables. Are you using NVM4W or nvm for Linux/macOS? They're different projects (this is the former).
